explorer/client/src/components/transaction-card/RecentTxCard.tsx

```diff
@@ -34,7 +33,7 @@ import type {
 import styles from './RecentTxCard.module.css';
 
 const TRUNCATE_LENGTH = 10;
-const NUMBER_OF_TX_PER_PAGE = 15;
+const NUMBER_OF_TX_PER_PAGE = 20;
```

Comment:
Just one small thing, can we keep this at 15? The Figma design on the home page is 15 transactions

Reply:
Looking at the deployed website, although the constant says '15', '20' items are displayed because the filter is overriding this constant. If we did enforce 15 per page this would potentially be confusing in that the filter says "20 per page". In the next update to the Home Page design, feel free to change this to 15 but make sure that this constant is not overridden.